addon/services/intl.d.ts

```diff
@@ -0,0 +1,95 @@
+import Evented from '@ember/object/evented';
```

Comment:
> In order for your addon's users to benefit from type information from your addon, you need to put .d.ts declaration files at the location on disk where the compiler expects to find them. This addon provides two commands to help with that: ember ts:precompile and ember ts:clean. The default ember-cli-typescript blueprint will configure your package.json to run these commands in the prepublishOnly and postpublish phases respectively, but you can also run them by hand to verify that the output looks as you expect.

Even if we convert all our public modules to ts that we would need to still maintain declaration files?  Am I reading this correct?